recipes-core/initscripts/initscripts-1.0/udev-volatiles.sh

```diff
@@ -3,25 +3,25 @@
 # Populate early tmpf that may be required by udev.
 #
 
-. /etc/default/rcS
-. /etc/init.d/functions
+set -e
```

Comment:
Is this `set -e` intentional?

Reply:
It is, although this one would only stop the initscript if `/etc/init.d/functions-selinux` is missing really.